src/methods/clientMethods.mjs

```diff
@@ -5,36 +5,62 @@ import Seance from '../models/Seance.mjs';
 import BlockedSeats from '../models/BlockedSeats.mjs';
 import User from '../models/User.mjs';
 import messages from '../namedMessages/namedMessages.mjs';
+import { filterSeancesByFeatures } from '../utils/utils.mjs';
 
 export async function getMovie(id) {
   const movie = await Movie.findById(id);
   return movie;
 }
 
+export async function getCurrentMovies() {
+  const currentDate = new Date();
+  currentDate.setHours(0, 0, 0);
+
+  try {
+    const movies = await Movie.find().where({ startDate: { $lte: currentDate }, endDate: { $gte: currentDate } });
+
+    if (movies) {
+      return movies;
+    }
+
+    return [];
+  } catch (error) {
+    console.error(error);
+  }
+}
+
 export async function getMovieSeances(params) {
   const { movieId, movieTheaterId, city, features, date } = params;
   const nowTime = new Date();
   const today = new Date(date);
 
+  const { emptyMoreOne, emptyMoreTwo, hasEmptyVip, hasEmptyDouble, video3d } = features;
```

Comment:
if it's boolean, try to name it more like boolean

Reply:
fixed